step.yml

```diff
@@ -1,26 +1,30 @@
 title: Set Xcode Project Build Number
 summary: |-
-  Set the value of your iOS app's bundle version in the `Info.plist` file to the specified version number.
+  Set the value of your iOS app's bundle version to the specified version number.
```

Comment:
Did you consider updating the step title too? I'm not an expert on all of these version-number-related fields, but the summary says bundle version and the title says build number.

Reply:
Good point. I modified it to use `build number` everywhere.